content/en/tracing/profiling/_index.md

```diff
@@ -219,6 +272,19 @@ Once enabled, the following profile types are collected:
 | Lock                     | Shows the time each function spent waiting for a lock.                                                                                                                                                                                                                                               |
 
 {{% /tab %}}
+
+{{% tab "Go" %}}
+
+Once enabled, the following profile types are collected:
+
+| Profile type             | Definition                                                                                                                                                                                                                                                                                         |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU                      | Shows the time each function spent running on the CPU.                                                                          |
+| Allocation               | Shows the amount of heap memory allocated by each function, including allocations which were subsequently freed. Go calls this `alloc_space`              |
+| Heap                     | Shows the amount of heap memory allocated by each function that remained allocated. Go calls this `inuse_space`                 |
```

Comment:
![image](https://user-images.githubusercontent.com/16651846/79785148-60352600-8311-11ea-8df7-d5d8eb8afc9c.png)

Reply:
counts shouldn't be there, but in summary table, shouldn't they ?

Reply:
Eventually, not planned for this sprint.